support/vs-code/silverlsp/README.md

```diff
@@ -0,0 +1,70 @@
+# silverlsp README
```

Comment:
Strictly speaking, one can run LSP with editors other than VS Code. Do you envision some other organization to reflect this? I know this is early days, but this may be worth considering.

Reply:
This is the VS code extension for Silver that makes use of the Silver language extension. I have named the extension `silverlsp`, it lives under `support/vs-code` for the moment.  The README file here is auto-generated and needs to be updated once this is beyond an early prototype.  The actual reusable language server implementation is under the `language-server` folder.